masterlist.yaml

```
@@ -21763,6 +21763,34 @@ plugins:
       - crc: 0x1BE50A99
         util: 'SSEEdit v4.0.4'
 
+  - name: 'RynsWhiterunCityLimits.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/65661/'
+        name: 'Ryn''s Whiterun City Limits'
+    msg:
+      - <<: *patch3rdParty
+        subs:
+          - 'eFPS - Exterior FPS boost'
+          - '[Unofficial eFPS patches](https://www.nexusmods.com/skyrimspecialedition/mods/70176/)'
+        condition: 'active("Occ_Skyrim_Tamriel.esp") and not active("Occ_Skyrim_RynsWhiterunCityLimits_patch")'
+      - <<: *patch3rdParty
+        subs:
+          - 'Ryn''s Bleakwind Basin'
+          - '[Ryn''s Skyrim Official Patch Hub](https://www.nexusmods.com/skyrimspecialedition/mods/73778/)'
+        condition: 'active("Ryn''s Bleakwind Basin.esp") and not active("Ryn''s_Whiterun_City_Limits_Ryn''s_Bleakwind_Basin_Patch.esp")'
+      - <<: *patch3rdParty
+        subs:
+          - 'Ryn''s Halted Stream Camp'
+          - '[Ryn''s Skyrim Official Patch Hub](https://www.nexusmods.com/skyrimspecialedition/mods/73778/)'
+        condition: 'active("Ryn''s Halted Stream Camp.esp") and not active("Ryn''s_Whiterun_City_Limits_Ryn''s_Halted_Stream_Camp_PATCH.esp")'
+      - <<: *patchProvided
+        subs:
+          - 'CC Tundra Homestead'
```

Single-element lists can be written in flow style:
```suggestion
        subs: [ 'CC Tundra Homestead' ]
```